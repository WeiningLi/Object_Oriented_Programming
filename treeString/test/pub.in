ra "Hello world"
^a6 "great "
pa
rb ", how are you?"
pb
+cab
pc

