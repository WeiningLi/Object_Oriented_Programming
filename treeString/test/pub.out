Hello great world
, how are you?
Hello great world, how are you?
